### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by lhy0403**
Enjoy!

### CODE REVIEW

File: translations/ru-RU.ts [hunk 1]
```diff
@@ -896,7 +896,7 @@
     <message>
       <location filename="../src/w_PrefrencesWindow.cpp" line="334"/>
       <source>tProxy is not supported on macOS and Windows</source>
-      <translation type="unfinished">tProxy is not supported on macOS and Windows</translation>
+      <translation>tProxy не поддерживается на MacOS и Windows</translation>
```
Comment (lhy0403) [inline translations/ru-RU.ts]: @thebadgateway macOS should start in lowercase

Comment (thebadgateway) [reply translations/ru-RU.ts]: I apologize, did not know that they are different. I read Wikipedia.

Comment (lhy0403) [reply translations/ru-RU.ts]: It's fine. I knew about that just a few days ago as well. We should thank @DuckSoft for #74

Comment (thebadgateway) [reply translations/ru-RU.ts]: xDD

Comment (lhy0403) [reply translations/ru-RU.ts]: Sir, is the capital letter issue now fixed?
v1.3.7.2 will be out these days, and this PR will be merged after a few tests.

Comment (thebadgateway) [reply translations/ru-RU.ts]: I'll be at the computer in an hour and fix it right away

Comment (lhy0403) [reply translations/ru-RU.ts]: Don't worry, It's midnight now in my country. I'll to that tomorrow or the day after.

**Pull request merged**